app/models/facility.rb

```diff
@@ -98,7 +98,7 @@ def facilities
   end
 
   def cohort_analytics(period, prev_periods)
-    query = CohortAnalyticsQuery.new(registered_hypertension_patients)
+    query = CohortAnalyticsQuery.new(assigned_patients)
```

Comment:
Are we including non-HTN patients by changing this to `assigned_patients`? We should preserve the HTN-only scoping we used to do, right?

Reply:
The scoping is preserved, will go ahead and change the name to `assigned_hypertension_patients`.